<?xml version="1.0" encoding="UTF-8"?><testrun name="TestCreatorJavaFileUsingJavaC" project="PADL Creator JavaFile (JavaC) Tests" tests="3" started="3" failures="1" errors="2" ignored="0">
  <testsuite name="TestSuite with 3 tests [example: padl.creator.javafile.javac.test.simple.Test1]" time="0.051">
    <testsuite name="padl.creator.javafile.javac.test.simple.Test1" time="0.051">
      <testcase name="testA" classname="padl.creator.javafile.javac.test.simple.Test1" time="0.033">
        <error>java.lang.IllegalAccessError: class jct.tools.JCTCreatorFromSourceCode (in unnamed module @0x36f6e879) cannot access class com.sun.tools.javac.api.JavacTool (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.api to unnamed module @0x36f6e879&#13;
	at jct.tools.JCTCreatorFromSourceCode.createJCT(JCTCreatorFromSourceCode.java:170)&#13;
	at padl.creator.javafile.javac.JavaFileCreator.create(JavaFileCreator.java:175)&#13;
	at padl.kernel.impl.CodeLevelModel.create(CodeLevelModel.java:41)&#13;
	at padl.creator.javafile.javac.test.simple.Test1.setUp(Test1.java:51)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:140)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:130)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
&#13;
</error>
      </testcase>
      <testcase name="testB" classname="padl.creator.javafile.javac.test.simple.Test1" time="0.012">
        <failure>java.lang.AssertionError: Class CreatorJava does not exist&#13;
	at org.junit.Assert.fail(Assert.java:89)&#13;
	at org.junit.Assert.assertTrue(Assert.java:42)&#13;
	at org.junit.Assert.assertNotNull(Assert.java:713)&#13;
	at padl.creator.javafile.javac.test.simple.Test1.testB(Test1.java:77)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:177)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:142)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:130)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
&#13;
</failure>
      </testcase>
      <testcase name="testC" classname="padl.creator.javafile.javac.test.simple.Test1" time="0.003">
        <error>java.lang.NullPointerException: Cannot invoke "padl.kernel.IFirstClassEntity.getConstituentFromID(char[])" because "entity" is null&#13;
	at padl.creator.javafile.javac.test.simple.Test1.testC(Test1.java:100)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:177)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:142)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:130)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:241)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:236)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
&#13;
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testrun>
